Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -263,12 +284,19 @@ - (void)testDefaultOnCrashedLastRun
     XCTAssertNil(options.onCrashedLastRun);
 }
 
+- (void)testGarbageOnCrashedLastRun_ReturnsNil
+{
+    SentryOptions *options = [self getValidOptions:@{ @"onCrashedLastRun" : @"fault" }];
+
+    XCTAssertNil(options.onCrashedLastRun);
+}
+
 - (void)testIntegrations
 {
     NSArray<NSString *> *integrations = @[ @"integration1", @"integration2" ];
     SentryOptions *options = [self getValidOptions:@{ @"integrations" : integrations }];
 
-    XCTAssertEqual(integrations, options.integrations);
+    [self assertArrayEquals:integrations actual:options.integrations];
```

Comment:
can we add a test to validate messed-up integrations? with String and non String values?

since it uses `filteredArrayUsingPredicate:isNSString`, we can test that

Reply:
same for inAppIncludes

Reply:
Good point, but they were already there:
https://github.com/getsentry/sentry-cocoa/blob/9e6c6e3dbb85bed13d056dc5dd77341823683544/Tests/SentryTests/SentryOptionsTest.m#L689-L701

and

https://github.com/getsentry/sentry-cocoa/blob/9e6c6e3dbb85bed13d056dc5dd77341823683544/Tests/SentryTests/SentryOptionsTest.m#L729-L737

Reply:
ups, my bad, thanks